### PR TITLE
docs(basics/schedule.md): translate

### PR DESCRIPTION
**Pull request by Azard**
##### Checklist

- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)

`docs/source/zh-cn/basics/schedule.md`
`docs/source/en/basics/schedule.md`

##### Description of change

update schedule.md and english translation

### CODE REVIEW

File: docs/source/en/basics/schedule.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+Although the HTTP Server we developed using the framework is a request-response model, there are still many scenarios that need to execute some scheduled tasks, for example:
+
+1. Regularly report server application status.
+1. Regularly update the local cache from the remote interface.
+1. Regularly split files, delete temporary files.
+
+The framework provides a mechanism to make the development and maintenance of scheduled tasks more elegant.
+
+## Develope Scheduled Tasks
```
Comment (atian25) [inline docs/source/en/basics/schedule.md]: `Develope`  typo

File: docs/source/en/basics/schedule.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+All scheduled tasks are stored in directory `app/schedule`. Each file is an independent scheduled task that could configure the properties and the detail methods to be executed.
```
Comment (atian25) [inline docs/source/en/basics/schedule.md]: how about change `are stored in` -> `should be placed in` ?

File: docs/source/en/basics/schedule.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+  * task(ctx) {
+    const res = yield ctx.curl('http://www.api.com/cache', {
+      dataType: 'json',
+    });
+    ctx.app.cache = res.data;
+  },
+};
+```
+
+Finished writing this file means that the definition of a scheduled task is complete. This scheduled task will be executed every 1 minute on every worker process, the requested remote data will be mounted back to `app.cache`.
```
Comment (atian25) [inline docs/source/en/basics/schedule.md]: `Finished writing this file means that the definition of a scheduled task is complete.` 感觉这句的中英文都可以干掉了。

File: docs/source/en/basics/schedule.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+### Schedule Mode
+
+Schedule tasks can specify `interval` or `cron` two different schedule mode.
+
+#### interval
+
+Configure the scheduled tasks by `schedule.interval`, scheduled tasks will be executed every specified time interval. `interval` can be configured as
+
+- Integer type, the unit is milliseconds, e.g `5000`.
+- String type, will be translated to milliseconds by [ms](https://github.com/zeit/ms), e.g `5s`.
```
Comment (atian25) [inline docs/source/en/basics/schedule.md]: `transform` is better?

File: docs/source/en/basics/schedule.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+    interval: '10s',
+  },
+};
+```
+
+#### cron
+
+Configure the scheduled tasks by `schedule.interval`, scheduled tasks will be executed at cron expressions specified timing. cron expressions are parsed by [cron-parser](https://github.com/harrisiirak/cron-parser).
+
+**Warning: cron-parser support second (linux crontab nonsupport).**
```
Comment (atian25) [inline docs/source/en/basics/schedule.md]: - `Warning` -> 'Note'
- which don't support by linux crontab

File: docs/source/en/basics/schedule.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+```
+
+When the application starts up, manually perform the scheduled tasks for system initialization, waiting for the initialization finished and then starting the application. See chapter [Application Startup Configuration](./app-start.md), we can implement initialization logic in `app.js`.
+
+```js
+module.exports = app => {
+  app.beforeStart(function* () {
+    // ensure the data is ready before the application start listening port
+    // follow-up data updates automatically by the scheduled task
+    yield app.runSchedule('update_cache');
```
Comment (atian25) [inline docs/source/en/basics/schedule.md]: @dead-horse 其实感觉这里有点牵强，schedule 的 task 一般应该就是调用一个 service 而已，没必要通过 `runSchedule` 执行

File: docs/source/en/basics/schedule.md (hunk 2 above)
Comment (Azard) [reply docs/source/en/basics/schedule.md]: agree

File: docs/source/en/basics/schedule.md (hunk 3 above)
Comment (Azard) [reply docs/source/en/basics/schedule.md]: agree

File: docs/source/en/basics/schedule.md (hunk 4 above)
Comment (Azard) [reply docs/source/en/basics/schedule.md]: agree

**Comment by Azard** [comment]
好像我从自己的 origin/master checkout 了，这个似乎合并会有问题，需要我重新开个 PR 从 upstream/master checkout 吗

**Comment by atian25** [comment]
不用新开，我直接 squash 好了.

**Comment by dead-horse** [comment]
@atian25 我合掉了

**Comment by atian25** [comment]
嗯，合吧，可以继续改为 async 了。

**Comment by atian25** [comment]
https://github.com/eggjs/egg/pull/1656

**Pull request merged**